fpdcc/ccfp-asset-dashboard
# Associate the asset tool with a phase

## Overview
See title. For issue #94. I also changed the LocalAsset.phase relationship to a `ForeignKey` instead of a `ManyToManyField`.

### Demo
![phase-assets](https://user-images.githubusercontent.com/38969506/151218007-c64d226b-5409-4d7c-b3bb-508bba2481d2.gif)


### Notes
This PR is still using the prototype's map UX. Those changes are to come with the relevant issues.

## Testing Instructions
- Visit review app: https://ccfp-asset-d-phase-asse-ffhjgx.herokuapp.com/
- Login with the "ccfp-asset-dashboard test user" credentials in the LastPass shared folder.
- Select a project and go to the phases page.
- Select a phase.
- Click "edit assets" in the top left corner.
- Search for some assets. Save them.
- Make sure the assets load.
- Leave the page and come back via the phase edit page. Make sure your saved assets load in the map.
